server/controllers/socket.controller.ts

```diff
@@ -1,17 +1,21 @@
 import type { NextFunction, OpineRequest, OpineResponse } from "../deps.ts";
+import { getCookies, verifyJWT } from "../deps.ts";
 import type { RoomSchema } from "../models/room.model.ts";
 import {
 	JoinEvent,
 	WebSocketEvent,
 	RoomUpdateEvent,
 	ConnectEvent,
 	OptionSelectedEvent,
+	Timeout,
 } from "../types/socket.ts";
 import calculateConfidence from "../utils/calculateConfidence.ts";
 import connectToDb from "../utils/connectToDb.ts";
 import { lookupRoom } from "../utils/db.ts";
+const JWT_SECRET = Deno.env.get("JWT_SECRET");
 
 const sockets = new Map<string, WebSocket>();
+const timeouts: Timeout[] = [];
```

Comment:
Would a `Map` make more sense here? Might be easier to manipulate with userId as the key.

Reply:
Good call, implementing.